index.md

```diff
@@ -15,6 +15,17 @@ tensors. A particular focus of Zarr is to provide support for storage
 using distributed systems like cloud object stores, and to enable
 efficient I/O for parallel computing applications.
 
+![arabidopsis](/images/arabidopsis.png "arabidopsis")
+<center>Image by <a href="https://www.openmicroscopy.org/2020/11/04/zarr-data.html">OME</a></center>
```

Comment:
I'll get you a blurb that includes the original author as well.

Also, what do you think perhaps about putting this into a banner at the top?

https://mmistakes.github.io/minimal-mistakes/layout/uncategorized/layout-header-overlay-image/

Reply:
e.g.:

Image courtesy of [OME](https://www.openmicroscopy.org/2020/11/04/zarr-data.html) by Valuchova _et al._ CC-BY 4.0, [eLife 2020](https://doi.org/10.7554/eLife.52546) ([idr0077](https://doi.org/10.17867/10000144))

Reply:
Done.